Documentation/ApiOverview/BackendModules/TemplateClass/BackendTemplateViewWithoutExtbase.rst

```diff
@@ -0,0 +1,144 @@
+.. include:: /Includes.rst.txt
+.. index:: ! JOCHEN
```

Comment:
why is Jochen here?

Reply:
🤦 Sorry, removed it. Was left over code from my tests.